src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -313,6 +313,7 @@ private ResourceManagementClient ResourceManagementClient
 
         protected internal void ValidateParameters()
         {
+            FindProxyModuleInstalled();
```

Comment:
I wonder if this is needed here, since `FindProxyModuleInstalled` is called where the path is needed and used?

Reply:
You are right. I just added it there temporarily for testing, and forgot to move it back. Thanks for pointing it out.